recipes-bsp/bootfiles/bcm2835-bootfiles.bb

```diff
@@ -34,6 +34,8 @@ do_deploy() {
     touch ${DEPLOYDIR}/${PN}/${PN}-${PV}.stamp
 }
 
+do_deploy[depends] += "rpi-config:do_deploy"
```

Comment:
I agree that the dependency is not clear enough but I'm actually thinking that this is not the actual consumer of the `config.txt`. The actual consumer is the image generation task `do_image_rpi_sdimg`. It would sound a little more reasonable to have this dependency on that [side](https://github.com/agherzan/meta-raspberrypi/blob/d34e054d5140792150d2a7ae3c0264eb254c8aa3/classes/sdcard_image-rpi.bbclass#L52). What do you think?

Reply:
I think it would make sense to have it in both places. If someone does bcm2835-bootfiles:do_deploy, then (IMO) rpi-config:do_deploy should still run. I see rpi-config as an implementation detail of bcm2835-bootfiles.

I have just committed a change that does this.

(Sorry for duplicate comment, I forgot to use Reply)